testing/sdk_networks.py

```diff
@@ -1,10 +1,18 @@
+import json
+
 import shakedown
 
 
 def check_task_network(task_name, expected_network_name="dcos"):
     """Tests whether a task (and it's parent pod) is on a given network
     """
     _task = shakedown.get_task(task_id=task_name, completed=False)
+
+    if type(_task) == list:
+        assert len(_task) == 1, "Found too many tasks matching {}, got {}"\
+            .format(task_name, _task)
+        _task = _task[0]
```

Comment:
No longer relevant, but definitely don't have the same variable be both a tuple and a tuple entry.